### PR TITLE
Fix schemaform inputs shrinking at large screen sizes

### PR DESCRIPTION
**Pull request by jbalboni**
Current:
screen shot 2018-02-28 at 9 19 40 am

Fixed:
screen shot 2018-02-28 at 9 19 48 am

### CODE REVIEW

File: src/sass/modules/_m-schemaform.scss [hunk 1]
```diff
@@ -14,9 +14,6 @@ fieldset {
   display: inline-block;
   width: 100%;
   max-width: 750px;
-  @media (min-width: $large-screen) {
```
Comment (jbalboni) [inline src/sass/modules/_m-schemaform.scss]: @cehsu I couldn't find anything that I broke by removing this, but do you remember what it was added for?

**Review by rianfowler — APPROVED** [review]

File: src/sass/modules/_m-schemaform.scss (hunk 1 above)
Comment (cehsu) [reply src/sass/modules/_m-schemaform.scss]: I'm not sure which specific screen size/browser/view scenario this addresses. The comment indicates this is to address an IE bug. It is part of a bug fix [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/7338) for the photo tool page and its view within the review page. I'll test this out and report back.

Comment (jbalboni) [reply src/sass/modules/_m-schemaform.scss]: I tested on IE and didn't notice any issues on the review page, at least.

**Comment by jbalboni** [comment]
I'm going to merge this and get it released, since it's affecting all forms. If we find a place where it's causing problems in VIC, we can fix that later.

File: src/sass/modules/_m-schemaform.scss (hunk 1 above)
Comment (cehsu) [reply src/sass/modules/_m-schemaform.scss]: The issue is on the normal cropper page, not the review page:
screen shot 2018-03-09 at 3 03 03 pm
An issue has been created for this bug [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9197).

**Pull request merged**